File: csvideo-view-interface/src/main/java/com/csvideo/view/service/UserLikeVideosService.java
package com.csvideo.view.service;

import com.csvideo.entity.PageResult;

/**
 * Created by asus on 2019/3/15.
 */
public interface UserLikeVideosService {
    /**
     * 点赞视频
     */
    public void add(String userId,String videoId);
    /**
     * 查找所有点赞视频
     */
    public PageResult findAll(String userId,int pageNum,int pageSize);
}

File: csvideo-view-web/src/main/webapp/js/service/UserLikeVideosService.js
//点赞短视频服务
app.service('UserLikeVideosService',function ($http) {
    //点赞
    this.add=function (userId,videoId) {
        return $http.get('../userLikeVideos/add?userId='+userId+'&videoId='+videoId);
    }
    //查找所有喜欢的视频
    this.getAll=function (userId,page,rows) {
        return $http.get('../userLikeVideos/getAll?userId='+userId+'&page='+page+'&rows='+rows);
    }
});